Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:9.5.0-alpine
```

Comment:
Is there any specific reason to use Node 9.x? Mainly just curious, I'd personally stick with LTS releases if there isn't a specific feature we are utilizing.

Reply:
We initially didn't update the Node version at all, @piehei updated it and we just updated our docker to match 🤷‍♂️

Reply:
No any real reason. I picked the latest at the time of the update and didn't think about the LTSness of it...

Reply:
We're using 8.9.4 (LTS) in our frontend repo, could probably use the same here?

Reply:
The LTS release is fine IMO, I don't think the 9.X branch has anything new and amazing in it, while 8.X added async / await support. Maybe just lock both package.json and the Dockerfile to Node 8 so we get minor and patch upgrades on every deploy